index.html

```diff
@@ -0,0 +1,798 @@
+    </p>
+    <p class="container box-1">
+      Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal
+      cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of
+      light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but
+      the universe might be the ultimate free lunch.”
+    </p>
+    <p class="container box-1">
+      Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest
+      that
+      inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion,
+      the
+      rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the
+      multiverse.
+
+
+    </p>
+    <p class="container box-1">
+      So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into
+      the past.
+    </p>
+    <p class="container box-1">We might never know what happened before inflation, at the very beginning, because inflation
+      erases everything that came before it. All the chaos and randomness of the primordial moment are swept away,
+      forever out of our view.
+
+    </p>
+    <p class="container box-1">
+      “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the
+      team, “you wind up at inflation.”
+    </p>
+    </div>
+
+    <div class="container extra-info">
+      <p>A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with
+        the <br> headline: Space Ripples Reveal Big Bang’s Smoking Gun. <a href="">Order Reprints</a> | <a
+          href="">Today’s Paper</a> | <a href="">Subscribe</a> </p>
+    </div>
+
+    <div class="container bottom-social-media-icons flex">
+      <button>READ 615 COMMENTS</button>
+
+      <ul class="bottom-ul-social-icons flex">
+        <li class="media-spacing"><i class="fab fa-facebook-f"></i></li>
+        <li class="media-spacing"><i class="fab fa-twitter"></i></li>
+        <li class="media-spacing"><i class="fas fa-envelope"></i></li>
+        <li class="media-spacing"><i class="fas fa-share"></i></li>
+      </ul>
+    </div>
+  
+<section class="grid-1">
+    <div class="grid-section-title">
+      <div>
+        <hr class="grid-section-title-hr-astronomy">
+        <h3 class="astronomy-title">More in space and astronomy</h3>
+      </div>
+      <div>
+        <hr class="gird-section-most-popular-hr">
+        <h3 class="most-popular-title">Most Popular</h3>
+      </div>
+
+    </div>
+
+    <!-- grid images section -->
+
+    <div class="grid-section">
+      <!-- space astronomy -->
+
+      <div class="space-astronomy">
+
+        <div class="grid-images-section">
+          <img src="images/related1.jpg" alt="">
```

Comment:
- [ ] **MANDATORY** All your images in this grid-section should be links as in the o riginal page